plugins/outputs/cloudwatch/cloudwatch.go

```diff
@@ -332,14 +371,17 @@ func (c *CloudWatch) pushMetricDatumBatch() {
 	}
 }
 
-//sleep some back off time before retries.
+// backoffSleep sleeps some amount of time based on number of retries done.
 func (c *CloudWatch) backoffSleep() {
 	var backoffInMillis int64 = 60 * 1000 // 1 minute
```

Comment:
Are there any reason why we need to keep up the 1 minute but not as low as possible? For example, 30s or 15s (even though the higher value of backOff, the more distribution Backoff would be but I'm still seeing 30s would be enough to cover the distribution?

Reply:
`backOffSleep()` starts with a low amount of sleep time (200 ms) and increases it.
I assume the intent is that the Agent never wait more than a minute between retries, anything above that is considered excessive. 

Why not set the max sleep to 30 seconds...?
There is no 1 right answer here. 30 seconds could be an acceptable upper limit.
I would prefer not changing that specific behavior just yet.

Keep in mind that number of (number_of_retries * sleep_time) will give the total time a request is retried, so lowering either of those numbers affects the overall retry time.